Server/Services/RatingService.cs

```diff
@@ -93,6 +106,7 @@ public enum GameResult
     public class LeaderboardPlayer
     {
         public string SteamId { get; set; }
+        public int Rank { get; set; }
```

Comment:
`GetLeaderboard` would return players with the rank `0`, which is kind of inconsistent